specs/upgrade/index.src.html

```diff
@@ -307,6 +307,13 @@ <h2 id="key-concepts">Key Concepts and Terminology</h2>
   :: A {{Request}} is said to be <strong>upgraded</strong> if it is rewritten
      to contain a URL with a {{URL/scheme}} of <code>https</code> or
      <code>wss</code>.
+  :  <dfn export>safely upgradable requests</dfn>
+  ::  If an origin does not depend on this document's
+      <code><a>upgrade-insecure-requests</a></code> mechanism, all requests to
+      it are considered <strong>safely upgradable requests</strong>.  If an
+      origin depends on the upgrade-insecure mechanism, only requests from
+      clients that send a <a>Prefer</a>: <a>return=secure-representation</a> header are
```

Comment:
Technically requests from old clients that do not have mixed content blockers could also be considered "safely upgradable requests", but it's probably best to hope they go away rather than encouraging UA sniffing to detect them.